### ISSUE TITLE
PRU - Stay's as OFN Partner after Odría's coup

### ISSUE THREAD

**Issue by Blanker389** [TNODevs/TNO-Bug-Tracker#3894]
### Operating System

Windows 10

### HOI4 version

1.15.3

### TNO version

1.7.0

### Do you have other mods in use?

None

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

TOA

### Files

[game.log](https://github.com/user-attachments/files/18233446/game.log)
[error.log](https://github.com/user-attachments/files/18233447/error.log)
[WRS_1962_01_01_01.zip](https://github.com/user-attachments/files/18233451/WRS_1962_01_01_01.zip)

### Describe the bug

After Odría's self-coup in Peru, they will join the Japanese economic sphere but remain as an OFN Partner.

### Steps to reproduce / Cause of the issue

1. Elect Odría and perform the coup.
2. Observe issue.

### Expected behaviour

Peru is no longer an OFN Partner.

### Screenshots

![image](https://github.com/user-attachments/assets/2aa8fa51-9685-4df9-92a4-e9fd15c4d345)

**Comment by Marok132** [comment]
I don't think it is a bug. Being in the another economic sphere does not mean you will leave the faction.

**Comment by Marok132** [comment]
> I don't think it is a bug. Being in another economic sphere does not mean you will leave the faction.
